mappings/83ad8607e830bf1ff5df2b55ae806256099e8d92cb896454588380ed50455045424c554520506170657268616e64.json

```diff
@@ -0,0 +1,54 @@
+{
+    "subject": "83ad8607e830bf1ff5df2b55ae806256099e8d92cb896454588380ed50455045424c554520506170657268616e64",
+    "url": {
+        "sequenceNumber": 0,
+        "value": "https://twitter.com/pepeblueada",
```

Comment:
Hi @CardanoTokenIO,

can you add policyID to the twitter bio? If it is not your account, you can submit without URL value.

Thanks!

Reply:
Hi, it's a clients token I am just providing the service!